src/test/java/nextstep/fp/LambdaTest.java

```diff
@@ -33,19 +33,25 @@ public void runThread() throws Exception {
 
     @Test
     public void sumAll() throws Exception {
-        int sum = Lambda.sumAll(numbers);
+        int sum = Lambda.sumWithConditional(numbers, (number) -> {
+            return true;
+        });
         assertThat(sum).isEqualTo(21);
     }
 
     @Test
     public void sumAllEven() throws Exception {
-        int sum = Lambda.sumAllEven(numbers);
+        int sum = Lambda.sumWithConditional(numbers, (number) -> {
+            return number % 2 == 0;
+        });
         assertThat(sum).isEqualTo(12);
     }
 
     @Test
     public void sumAllOverThree() throws Exception {
-        int sum = Lambda.sumAllOverThree(numbers);
+        int sum = Lambda.sumWithConditional(numbers, (number) -> {
+            return number > 3;
+        });
```

Comment:
return을 제거하고 사용할 수도 있겠네요! 
```suggestion
        int sum = Lambda.sumWithConditional(numbers, number -> number > 3);
```